examples/lustre/test.bats

```diff
@@ -0,0 +1,76 @@
+load "${CHTEST_DIR}/common.bash"
+
+setup () {
+    scope full
+    prerequisites_ok lustre
+
+    if [[ $SLURM_JOB_ID ]]; then
+        if [[ -z $ch_lustre ]]; then 
+            pedantic_fail 'no lustre to bind mount'
+        fi
+    else
+        if [[ -z $ch_lustre ]]; then
+            skip 'no lustre to bind mount'
+        fi
+    fi
+
+    # Check lustre directory exists
+    if [[ ! -e $ch_lustre ]]; then
+        echo "${ch_lustre} does not exist"
+        exit 1
+    fi
+    # Check lustre directory is a directory
+    if [[ ! -d $ch_lustre ]]; then
+        echo "${ch_lustre} is not a directory"
+        exit 1
+    fi
+
+    binds=${ch_lustre}:/mnt/0
+    work_dir=/mnt/0/charliecloud_testing_directory
+    mkdir -p "${ch_lustre}/charliecloud_testing_directory"
+}
+
+@test "${ch_tag}/create dir" {
+    ch-run -b "$binds" "$ch_img" -- mkdir "${work_dir}/tryme"
+}
+
+@test "${ch_tag}/create file" {
+    ch-run -b "$binds" "$ch_img" -- touch "${work_dir}/test_create.txt"
+}
+
+
+@test "${ch_tag}/delete file" {
+    ch-run -b "$binds" "$ch_img" -- rm -f "${work_dir}/test_create.txt"
+}
+
+@test "${ch_tag}/write" {
+    ch-run -b "$binds" "$ch_img" -- sh -c "sleep 5 && echo hello | dd of=${work_dir}/test_write.txt"
```

Comment:
Explain why you are sleeping and why you're using `dd`.

IIRC, you're using `dd` to avoid the disk cache? If so, use an explicit option for that.